src/plugins/extension-manager/index.tsx

```diff
@@ -0,0 +1,223 @@
+import * as React from "react";
+import ReactDOM from "react-dom";
+import styles from "./styles.less";
+import Tooltip from "components/Tooltip";
+
+import ExtensionManagerIcon from "assets/icon--extension-manager.svg";
+import TrashcanIcon from "assets/icon--trashcan.svg";
+import ExpansionBox, { ExpansionRect } from "components/ExpansionBox";
+import DraggableIcon from "assets/icon--draggable.svg";
+
+import { defineMessage } from "@formatjs/intl";
+import useStorageInfo from "hooks/useStorageInfo";
+
+const messages = defineMessage({
+  title: {
+    id: "plugins.ExtensionManager.title",
+    defaultMessage: "Extension Manager",
+  },
+  intro: {
+    id: "plugins.ExtensionManager.intro",
+    defaultMessage:
+      "Manage your extensions",
+  },
+});
+
+const DEFAULT_CONTAINER_INFO = {
+  width: 300,
+  height: 450,
+  translateX: 72,
+  translateY: 60,
+};
+
+interface ExtensionManager {
+  _loadedExtensions: Map<string, string>;
+  deleteExtensionById: (id: string) => void;
+}
+
+interface VirtualMachineWithExtensions extends VirtualMachine {
+  extensionManager: ExtensionManager;
+}
+
+const ExtensionManager: React.FC<PluginContext & { vm: VirtualMachineWithExtensions }> = ({ intl, utils, vm, redux }) => {
+  //console.log(vm)
+  //console.log(redux)
+
+  const locale = (redux.state.locales as any).locale
+
+  //patch the built-in getExtensionInfoByID
+  const getExtensionInfoById = (id: string) => {
```

Comment:
This still doesn't perfectly solve the issue of translating the names of built-in plugins.

Reply:
> This still doesn't perfectly solve the issue of translating the names of built-in plugins.

You can now get all extension information through the getAllExtensionInfo method in utils.